lib/ffmpeg/encoding_options.rb

```diff
@@ -9,27 +9,54 @@ def initialize(options = {}, prefix_options = {})
 
     def to_s
       params = collect do |key, value|
-        send("convert_#{key}", value) if value && supports_option?(key)
+        attempt_self_call(key, value)
       end
 
       prefix_params = @prefix_options&.map do |key, value|
-        send("convert_#{key}", value) if value && supports_option?(key)
+        attempt_self_call(key, value)
       end
 
       # codecs should go before the presets so that the files will be matched successfully
       # all other parameters go after so that we can override whatever is in the preset
-      input   = params.select { |p| p =~ /^\-i / }
+      inputs   = params.select { |p| p =~ /\-i / }
```

Comment:
nit: this breaks the nicely lined up `=` 😅 
```suggestion
      inputs  = params.select { |p| p =~ /\-i / }
```